src/libaudqt/url-opener-qt.cc

```diff
@@ -57,6 +57,7 @@ static QDialog * buildUrlDialog(bool open)
 
     auto dialog = new QDialog;
     dialog->setWindowTitle(title);
+    dialog->setWindowRole("history");
```

Comment:
Use `url-dialog` here please. Only its checkbox has a connection to a `history`.